src/platform/ESP32/NetworkProvisioningServerImpl.cpp

```diff
@@ -310,7 +311,7 @@ void NetworkProvisioningServerImpl::HandleScanDone()
 
         // Send the scan results to the requestor.  Note that this method takes ownership of the
         // buffer, success or fail.
-        err = SendNetworkScanComplete(encodedResultCount, std::move(respBuf));
+        err = SendNetworkScanComplete(encodedResultCount, writerStore.Release());
```

Comment:
```suggestion
        err = SendNetworkScanComplete(encodedResultCount, writer.Release());
```
I would think.

Reply:
Indeed. Looks like nothing builds this file yet.